src/NServiceBus.Core/IHandleMessages.cs

```diff
@@ -1,13 +1,11 @@
 namespace NServiceBus
 {
     using System.Threading.Tasks;
-    using JetBrains.Annotations;
 
     /// <summary>
     /// Defines a message handler.
     /// </summary>
     /// <typeparam name="T">The type of message to be handled.</typeparam>
-    [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
```

Comment:
Looks like the reason for the new inspection failures is because of this removal. We should be able to add more targeting comment-based exclusions for the tests though, right?

Reply:
@bording agreed. i pushed an update